orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/StartStageHandler.kt

```diff
@@ -76,6 +76,7 @@ open class StartStageHandler
             val exceptionDetails = exceptionHandlers.shouldRetry(e, stage.getName())
             if (exceptionDetails?.shouldRetry ?: false) {
               log.warn("Error planning ${stage.getType()} stage for ${message.executionType.simpleName}[${message.executionId}]")
+              message.setAttribute(MaxAttemptsAttribute(5))
```

Comment:
Yep. That's deliberate otherwise you get a message in a superposition of states.